cmd/opm/alpha/bundle/validate.go

```diff
@@ -50,7 +50,7 @@ func validateFunc(cmd *cobra.Command, args []string) error {
 		}
 	}()
 
-	err = imageValidator.PullBundleImage(tagBuildArgs, dir)
+	err = imageValidator.PullBundleImage(imageBuilderArgs, tagBuildArgs, dir)
```

Comment:
This change dissects pull image and validate image. Because we are making a directory that can be validated by this validator but the old implementation does not allow validation unless we let it pull from somewhere.